### PR TITLE
Move from FAU-inf2 to fau-fablab

### PR DESCRIPTION
**Pull request by mgmax**
- Travis now works everywhere, only uses the key if it is available for encryption.
- change app-common repo (obsoletes #9)

Can please someone test if the app still builds and works in reality? (Remove the old ../fablab-common folder before)

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -157,7 +157,7 @@ afterEvaluate {
     project.tasks.matching { it.name.startsWith('jaxRs2Retrofit') }.each {
         it.inputDir = file(project(':../fablab-common').projectDir.toString() + '/src/main/java')
         it.outputDir = generatedSourcesDir
-        it.packageName = 'de.fau.cs.mad.fablab.rest.myapi'
+        it.packageName = 'de.fau.fablab.app.rest.myapi'
```
Comment (patkan) [inline app/build.gradle]: I am not 100% into packaging on Android, but we might be braking installation with this.

Comment (patkan) [inline app/build.gradle]: OK, here is a source: http://stackoverflow.com/a/17582393
We would thereby create a NEW app, that cannot be installed "over" the old one.

Comment (patkan) [inline app/build.gradle]: Ah, I reread the complete code. In the excerpt I misstook it for changing the packagename.
But now I think this should be OK.

**Comment by patkan** [comment]
Apart from the build system being horribly whacky (I don't really like the way that fablab-common is included).
I have several problems building the project with this new PR and I don't really understand why the travis-build goes smoothly.

**Comment by patkan** [comment]
OK, I understand it now, some details:
Building on the commandline fails, I have probably a a bit messed up setup.
Building in Android Studio builds also the JaxRS-Stuff which horribly falls apart with the "split" namespace.

I think we should completely rename all of the app and package or not rename anything. Since we can't do the first one, we should revert the change of namespace in app-common. @jneureuther you changed it, was there a particular need?

**Comment by patkan** [comment]
@fau-fablab/app If no one opposes I will revert the namespace change (in fablab-common), update this PR and then merge if it is building.

**Comment by sedrubal** [comment]
@patkan that would be great :+1:

**Pull request merged**